src/tests/reactBatching.ts

```diff
@@ -0,0 +1,2 @@
+import { unstable_batchedUpdates } from 'react-native'
+export { unstable_batchedUpdates }
```

Comment:
pour info, il est possible de l'écrire comme ça

```suggestion
export { unstable_batchedUpdates } from 'react-native'
```